blib/Perl6/BOOTSTRAP/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Don't remove this file and dir.
+# They are needed to host the bytecode files during the build process
+for any VM.
+*.pbc
```

Comment:
This was "parrot byte code", which is no more. But maybe we want to ignore `*.maorvm`?

Reply:
Makes sense, will fix both.